src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/Seo/YoastHandler.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Frontend\Handlers\Seo;
+
+use StoreKeeper\WooCommerce\B2C\Exceptions\WordpressException;
+use StoreKeeper\WooCommerce\B2C\Frontend\Handlers\Seo;
+use StoreKeeper\WooCommerce\B2C\Options\StoreKeeperOptions;
+use StoreKeeper\WooCommerce\B2C\Tools\WordpressExceptionThrower;
+
+class YoastHandler implements SeoHandlerInterface
+{
+    const YOAST_SEO_OPTION_KEY = 'wpseo_taxonomy_meta';
+
+    public function handle($markdown, $product)
+    {
+        if (!self::isSelectedHandler()) {
+            // revert handler to storekeeper
+            StoreKeeperOptions::set(StoreKeeperOptions::SEO_HANDLER, Seo::STOREKEEPER_HANDLER);
```

Comment:
i think it should be in setting not on the showing of the page.

Reply:
Removed this part.